app/views/article/_show_default.html.erb

```diff
@@ -8,21 +8,34 @@
     <dl class="dl-horizontal dl-invert">
       <% blacklight_config.show.sections.each do |section_name, fields| -%>
         <% if (section = render('article/record/section_fields', document: document, fields: fields.keys)).present? -%>
-          <div class='section'>
-            <% if section_name != 'Summary' -%>
-              <div class='section-heading'>
-                <% mini_map_sections[section_name.downcase] = true %>
-                <h2 id="<%= section_name.downcase %>">
-                  <%= t("record_side_nav.#{section_name.downcase}.icon").html_safe %>
```

Comment:
Why did we remove the icons in this PR?  It doesn't seem to be explicitly specified in the ticket that this closes.